cloud/test/fdb_injection_test.cpp

```diff
@@ -67,6 +67,7 @@ int main(int argc, char** argv) {
 
     cloud::config::enable_txn_store_retry = true;
     cloud::config::txn_store_retry_times = 100;
+    cloud::config::txn_store_retry_base_intervals_ms = 1;
```

Comment:
1ms may be too short to prevent an avalanche

Reply:
This test is mainly used to test whether ms proxy successfully retries these errors.